docker-images/syntax-highlighter/crates/scip-syntax/testdata/example.c

```diff
@@ -1,3 +1,78 @@
-int main() {
-  return 0;
+#include <stdlib.h>
+#include <sys/types.h>
+
+#define BUFSIZE 4096
+
+const char *name = "com.horsegraph.connection";
+const char *author = "Petri & Thorsten";
+const int count;
+int count2;
+const int age = 28;
+static uint sweet_sweet_numbers[5] = {23, 420, 69, 42, 7};
+
+enum { BLACK, RED };
+
+enum animal {
+  ANIMAL_TOUCAN,
+  ANIMAL_TIGER,
+  ANIMAL_HORSE,
+  ANIMAL_GIRAFFE,
+  ANIMAL_GOPHER,
+  ANIMAL_ORANGUTAN
```

Comment:
It'd be more helpful to test different bits of syntax compared to adding more examples for the same kind of syntax.

```suggestion
  ANIMAL_TIGER = 1,
  ANIMAL_TIGGER = ANIMAL_TIGER,
```

Reply:
makes sense!